PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebVaultRequest.kt

```diff
@@ -6,7 +6,16 @@ package com.paypal.android.paypalwebpayments
  * @property [setupTokenId] ID for the setup token associated with the vault approval
  * @property [approveVaultHref] URL for the approval web page
  */
-data class PayPalWebVaultRequest(
+data class PayPalWebVaultRequest @Deprecated("Use PayPalWebVaultRequest(setupTokenId:) instead.")
```

Comment:
```suggestion
data class PayPalWebVaultRequest @Deprecated("Use PayPalWebVaultRequest(setupTokenId) instead.")
```
nit: Removing the colon. There's no set format that I can find for referencing the alternative constructor, but technically it'd be more like `PayPalWebVaultRequest(setupTokenId=)` in Kotlin.